nimbus/src/persistence.rs

```diff
@@ -15,6 +15,9 @@ use rkv::StoreOptions;
 use std::fs;
 use std::path::Path;
 
+const DB_KEY_DB_VERSION: &str = "db_version";
+const DB_VERSION: u16 = 1; // Increment and implement a DB migration in `maybe_upgrade` when necessary.
```

Comment:
Suggest we put a comment on the structs and enums that will cause us to increment this number and write a migration.

```rust
// Warning adding/removing to this enum will require incrementing DB_VERSION and a migration.
```